### PR TITLE
Add SRS Stages

### PR DESCRIPTION
**Pull request by vietqhoang**
Changes proposed in this pull request:

* Add documentation about /srs_stages

---

The credentials to view the Netlify deploy is the following:

* **username**: crabigator
* **password**: noblowfishallowed

### CODE REVIEW

File: source/includes/20170710/resources/_srs_stages.md.erb [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# SRS Stages
+
+SRS Stages summarize the space repetition intervals. These intervals are used to calculate the `assignment.available_at`.
```
Comment (oldbonsai) [inline source/includes/20170710/resources/_srs_stages.md.erb]:
```suggestion
This report summarizes the space repetition intervals. These intervals are used to calculate the `assignment.available_at`.
```

File: source/includes/20170710/resources/_srs_stages.md.erb [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+## Review Statistic Data Structure
```
Comment (oldbonsai) [inline source/includes/20170710/resources/_srs_stages.md.erb]:
```suggestion
## SRS Stages Data Structure
```

File: source/includes/20170710/resources/_srs_stages.md.erb [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+```
+
+Attribute | Data Type | Description
+--------- | --------------- | -----------
+accelerated_interval | Integer | The duration of the stage in seconds for levels which have accelerated space repetition.
+interval | Integer | The duration of the stage in seconds.
+srs_stage | Integer | The stage identifier.
+srs_stage_name | String | The name of the stage.
+
+## Get User Information
```
Comment (oldbonsai) [inline source/includes/20170710/resources/_srs_stages.md.erb]:
```suggestion
## Get the SRS Stages
```

**Comment by oldbonsai** [comment]
We should also change the title in https://github.com/tofugu/wanikani-api-v2-doc/blob/master/source/includes/20170710/_additional_information.md to something else — there's an anchor link collision, so the left navigation for the endpoint takes you to the additional information section. Maybe we call the additional information one 'Our SRS … System' or 'SRS in Action' or something.

**Comment by vietqhoang** [comment]
@oldbonsai I am not following. Where is the anchor link collision happening?

**Pull request merged**